### PR TITLE
FEATURE: invite page tabs

### PR DESCRIPTION
**Pull request by techAPJ**
Screenshots:

Reedeemed tab:

screen shot 2015-07-12 at 18 09 00

Pending tab:

screen shot 2015-07-12 at 18 09 14

### CODE REVIEW

**Comment by SamSaffron** [comment]
looks good @eviltrout can you review?

**Comment by eviltrout** [comment]
Already on my list for tomorrow first thing.

**Comment by techAPJ** [comment]
@eviltrout: I just updated the code to use `admin-nav-item` component for invitation tabs, and renamed it to `nav-item`.

File: app/assets/javascripts/discourse/controllers/invite.js.es6 [hunk 1]
```diff
@@ -132,14 +132,14 @@ export default ObjectController.extend(Presence, ModalFunctionality, {
...
-            userInvitedController = this.get('controllers.user-invited');
+            userInvitedController = this.get('controllers.user-invited-show');
 
       this.setProperties({ saving: true, error: false });
 
       return this.get('model').createInvite(this.get('emailOrUsername').trim(), groupNames).then(result => {
               this.setProperties({ saving: false, finished: true });
               if (!this.get('invitingToTopic')) {
-                Discourse.Invite.findInvitedBy(Discourse.User.current()).then(invite_model => {
+                Discourse.Invite.findInvitedBy(Discourse.User.current(), userInvitedController.get('filter')).then(invite_model => {
```
Comment (eviltrout) [inline app/assets/javascripts/discourse/controllers/invite.js.es6]: minor: should be `this.currentUser` instead of `Discourse.User.current()`

**Comment by eviltrout** [comment]
Looks really good!

File: app/assets/javascripts/discourse/controllers/invite.js.es6 (hunk 1 above)
Comment (techAPJ) [inline app/assets/javascripts/discourse/controllers/invite.js.es6]: Done via https://github.com/discourse/discourse/commit/fcdb7f36b7876e193a73d492a9ba115bc2966838 :)

**Pull request merged**